packages/slate/src/models/history.js

```diff
@@ -53,6 +53,24 @@ class History extends Record(DEFAULTS) {
     )
   }
 
+  /**
+   * Create a list of `Operations` from `operations`.
+   *
+   * @param {Array<Object>|List<Object>} operations
+   * @return {List<Object>}
+   */
+
+  static createList(operations = []) {
+    if (List.isList(operations) || Array.isArray(operations)) {
+      const list = new List(operations)
```

Comment:
probably not a huge performance issue, but could this be:

```
if (List.isList(operations)) {
  return list
} else if (Array.isArray(operations)) {
  return new List(operations)
}
// error
```

Reply:
@jtadmor I agree but I was conforming to existing identical constructs within the code as the PR template commanda

Reply:
fwiw I'd be happy to switch it over to the above ^

Reply:
based on things like https://github.com/ianstormtaylor/slate/blob/b26535fd947cb54153800799a1dfc5d4b7729218/packages/slate/src/models/block.js#L65-L74

Reply:
I went ahead and did this based on your suggestion